manifests/plugin/pulp/params.pp

```diff
@@ -3,4 +3,5 @@
 class foreman_proxy::plugin::pulp::params {
   $pulpcore_api_url     = "https://${facts['networking']['fqdn']}"
   $pulpcore_content_url = "${pulpcore_api_url}/pulp/content"
+  $rhsm_url             = "https://$facts['networking']['fqdn']}/rhsm"
```

Comment:
I chose this default to match the default for `pulpcore_mirror => false`

When `pulpcore_mirror => true` we would need to override this to `"https://$facts['networking']['fqdn']:$foreman_proxy_content::reverse_proxy::port/rhsm"` with the reverse proxy port